source/MaterialXTest/ShaderValid.cpp

```diff
@@ -455,6 +456,16 @@ static void runGLSLValidation(const std::string& shaderName, mx::TypedElementPtr
             validator.save(fileName);
 
             validated = true;
+
+            // REMOVE ME!
+            // Dump shader stages on error
+            std::ofstream file;
```

Comment:
I think we could keep this here, always dumping the shader source, not only on errors.

Reply:
We could but it will slow things down a bunch given how many tests we have.

Reply:
Maybe have an option for it then. I find it very useful to be able to look at the generated source code even when it's not failing to compile.
But it's fine to leave this for another change list.

Reply:
I'll remove this for now, looks like Bernard is working on something related to this.